src/client/Microsoft.Identity.Client/ApiConfig/AbstractAcquireTokenParameterBuilder.cs

```diff
@@ -347,6 +348,22 @@ public T WithB2CAuthority(string authorityUri)
             return this as T;
         }
 
+        /// <summary>
+        /// Specifies additional parameters acquired from authentication responses to be cached with the access token that are normally not included in the cache object.
+        /// </summary>
+        /// <param name="cacheParameters">Additional parameters to cache</param>
+        /// <returns></returns>
+        public T WithAdditionalCacheParameters(IEnumerable<string> cacheParameters)
+        {
+            if (cacheParameters != null && cacheParameters.Count() == 0)

```

Comment:
nit: Keep it flexible, just ignore the params if empty.

Reply:
ok